central/complianceoperator/v2/report/manager/manager_impl.go

```diff
@@ -188,7 +189,7 @@ func (m *managerImpl) generateReportNoLock(req *reportRequest) {
 		profiles = append(profiles, profile.GetProfileName())
 	}
 
-	repRequest := &reportGen.ComplianceReportRequest{
+	repRequest := &types.ComplianceReportRequest{
```

Comment:
Not a blocker.

Actually, the `complianceReportgenerator.Request` would read a bit better, wouldn't it?

Ideally (go idiomatic), we could call it `complianceReport.Request`, so that it is clear that this is a request to something in the package `complianceReport`. I am only not sure whether other usages of that package would fit to this naming convention.

Calling the package `types` feels a bit too generic to me.

Reply:
I think `complianceReport.Request` reads better but I'm not sure what to do about the folder structure.

If we change the `types` folder to `complianceReport` the structure would be something like:
```
- complianceReportgenerator
  └ compliancereport
```
Right now is:
```
- complianceReportgenerator
  └ types
```
I think the folder structure reads better with `types`.

The way I see it we have three options:
1. We change the package name and the folder name.
2. We change the package name but not the folder.
3. We alias the import to `complianceReport`. 
 
WDYT?

Reply:
You could go further with the structure if you like:

```
- complianceoperator
  - v2
    - report
      - manager
        - generator
        - format
```

ComplianceReportGenerator is incredibly redundant here.  I also understand if that is an undertaking you would rather not do

Reply:
I've opened a follow-up PR to address this here #13632